### DISCUSSION TITLE
How to increase context size (height) for coc-references?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [neoclide/coc.nvim#3936] [Q&A] [answered]
10 lines is too small, maybe by default should be 20.

I tried `let g:coc_config_list_height = 30` but the height still the same.

I do not want to use a JSON config file - for Vim I use `.vim` config files, and for TypeScript: `ftplugin/typescript.vim`.

![image](https://user-images.githubusercontent.com/3514015/180199823-2d95a844-3a87-4039-a231-3fba10029789.png)

**Comment by fannheyward** [comment]
It's `list.height` in coc-settings.json. You can config in your vimrc without `coc-settings.json`, checkout `:h coc#config()`.

**Reply by vitaly-zdanevich** [reply]
`:h coc#config()` does not work for me - what should I add to my vim config?

**Reply by fannheyward** [reply]
```vim
call coc#config('list', { 'height': 20 })
```

**Reply by vitaly-zdanevich** [reply]
Added, restarted Vim, opened TypeScript file, `gr` for `coc-references` still the same 10 lines :(

**Comment by vitaly-zdanevich** [comment, marked as answer]
`call coc#config('list', { 'maxPreviewHeight': 50 })`
